### PR TITLE
Updates fedora-31 -> fedora-32 throughout

### PR DESCRIPTION
**Pull request by conorsch**
Updates all text, but doesn't include new screenshots yet.

Mirrors the changes in https://github.com/freedomofpress/securedrop-workstation/pull/642. Depending on when Qubes 4.0.4 is released (https://github.com/freedomofpress/securedrop-workstation/issues/640), these docs may get stale fast, but they should be merged in tandem with support in https://github.com/freedomofpress/securedrop-workstation/pull/642.

Marking as draft because the screenshots need to be updated still.

### CODE REVIEW

File: docs/admin/upgrading_to_fedora_32.rst [hunk 1]
```diff
@@ -45,31 +45,31 @@ TEMPORARY: Update the ``qubes-mgmt-salt-vm-connector`` package
 
 .. important:: Due to an `upstream issue <https://github.com/QubesOS/qubes-issues/issues/6188>`_, blocking updates
```
Comment (zenmonkeykstop) [inline docs/admin/upgrading_to_fedora_32.rst:46]: We can probably remove this now - the fix is out.

**Review by zenmonkeykstop — CHANGES_REQUESTED** [review]
Couple small changes -I'm going to append a commit with any updated screenshots so I'm happy to make those edits too.

**Comment by zenmonkeykstop** [comment]
Have added affected screenshot so this is ready for review.

File: docs/admin/upgrading_to_fedora_32.rst [hunk 2]
```diff
@@ -16,60 +16,37 @@ Fedora 30 template.
 
 As of May 26, 2020, Fedora 30 templates are end-of-life. If you are
 provisioning SecureDrop Workstation for the first time, you will need to update
-your Fedora template manually from Fedora 31 *before* installing SecureDrop
+your Fedora template manually from Fedora 32 *before* installing SecureDrop
```
Comment (rocodes) [inline docs/admin/upgrading_to_fedora_32.rst:19]: > you will need to update your Fedora template manually from Fedora 32
 
you will need to update your Fedora template manually _to_ Fedora 32?

**Review by rocodes — DISMISSED** [review]
We can nitpick prepositions later (or never)!  Approving + merging since this is so timely.

**Review by rocodes — APPROVED** [review]
:100:

**Review by zenmonkeykstop — APPROVED** [review]

**Pull request merged**